templates/mainapp/layout.html

```diff
@@ -125,7 +125,16 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
                             <p id="bookauthors">
                             </p>
 
-                            <div id="bookrating"></div>
+                            {% if user.is_authenticated %}
+                            <div class='rating-block-modal' id='rating-block-' onmouseover="startRatingModal(this)" 
+                            onclick='rateBookModal(this)' onmouseout='finishRatesModal(this)'>
+                                <div id='rateYo' class="rate-" style="display: block"></div>
+                            </div>
+                            {% else %}
+                            <div class="prompt-login">
```

Comment:
For consistency, the behaviour when not logged in should be similar to what happens on other pages

Reply:
You mean that I should add this code (copied from `recommendations.html`)

```
<div class='rating-block btn' onclick="location.href='{% url 'account_signup' %}'">
    <div id='button-text'>Rate <i class="fa fa-heart" style="color:red"></i> This Book</a>
    </div>
</div>
```

Reply:
Yes

Reply:
![Result](https://user-images.githubusercontent.com/40713709/112143667-39a4fc80-8bfe-11eb-906b-9f6d10365b01.png)
It is looking something like this

Reply:
Yes